src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -175,7 +175,13 @@ private bool StartNewModelComputation(CompletionService completionService, Compl
 
             this.sessionOpt = new Session(this, computation, Presenter.CreateSession(TextView, SubjectBuffer, null));
 
-            sessionOpt.ComputeModel(completionService, trigger, _roles, GetOptions());
+            var options = GetOptions();
+            if (_isDebugger)
+            {
+                options = options.WithChangedOption(EditorCompletionOptions.UseSuggestionMode, true);
```

Comment:
I'm confused by teh title of this PR and this line.  Isn't this basically always setting suggestion mode when you're in the debugger?